portability-types-common/src/main/java/org/datatransferproject/types/common/models/music/MusicPlaylist.java

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright 2022 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.types.common.models.music;
+
+import static com.google.common.base.Strings.isNullOrEmpty;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.google.common.base.MoreObjects;
+import java.time.Instant;
+import java.util.Objects;
+
+/**
+ * POJO for https://schema.org/MusicPlaylist
+ */
+public class MusicPlaylist {
+
+  private final String id;
+  private final String title;
+  private final String description;
+  private final Instant createTime;
+  private final Instant updateTime;
+
+  @JsonCreator
+  public MusicPlaylist(
+      @JsonProperty("id") String id,
+      @JsonProperty("title") String title,
+      @JsonProperty("description") String description,
+      @JsonProperty("createTime") Instant createTime,
+      @JsonProperty("updateTime") Instant updateTime) {
+    if (isNullOrEmpty(id)) {
+      throw new IllegalArgumentException("id must be for MusicPlaylist");
+    }
+    this.id = id;
+    this.title = title;
+    this.description = description;
+    this.createTime = createTime;
```

Comment:
I see you're going to be more granular than just "date," but since hte name doesn't match the upstream spec, two suggestions:

1. add a tiny bit of javadoc to _just_ these fields clarifying how they're different - eg: even reference upstream like _"the same as https://schema.org/dateCreated but of uses java.time.Instant for arbitrary granularity usage"_ - this way the upstream doc can otherwise still be leaned on.
2. _slightly_ rename them to clarify it's only the format that's changed; eg: change from `{create,update}Time` `time{Create,Update}d` to more closely match upstream. (again just continuing that readbility-hint that like (1) that indicates this is intended to be the same fields one sees in the schema we link to).

---

i'm pasting from the schema.org page you link to in your javadoc above for quick-reference:

`dateCreated`	Date  or
DateTime	The date on which the CreativeWork was created or the item was added to a DataFeed.
`dateModified`	Date  or
DateTime	The date on which the CreativeWork was most recently modified or when the item's entry was modified within a DataFeed.

Reply:
Added the javadoc in latest commit https://github.com/google/data-transfer-project/pull/1131/commits/0ada90d3d2f133187ea2df70db866b5beb6ffffb.